### PR TITLE
Fix deepsource issues - PropTypes

### PR DESCRIPTION
**Pull request by RedFox0x20**
Added `propTypes` to `PrimaryLayout`

### CODE REVIEW

**Comment by RedFox0x20** [comment]
Note: Allow @ozer619 to review this PR to allow them to gain experience with using the PR review tools before merging.

File: components/PrimaryLayout.js [hunk 1]
```diff
@@ -27,3 +28,7 @@ export default function PrimaryLayout({ children }) {
     </>
   );
 }
+
+PrimaryLayout.propTypes = {
+  children: PropTypes.node,
```
Comment (marekrozmus) [inline components/PrimaryLayout.js:33]: maybe we could have this `isRequired`?

**Review by marekrozmus — APPROVED** [review]

File: components/PrimaryLayout.js (hunk 1 above)
Comment (RedFox0x20) [reply components/PrimaryLayout.js:33]: Left as not required as there isn't always a card available for some sections, particularly if they're in-dev, for these sections we still need to display the rest of the layout.

**Comment by RedFox0x20** [comment]
**Closed**
Was also changed in #127

**Pull request closed without merging**